tests/test_masked_loss.py

```diff
@@ -54,14 +54,12 @@ def test_shape(self, input_param, expected_val):
         pred = torch.randn(size)
         result = MaskedLoss(**input_param)(pred, label, None)
         out = result.detach().cpu().numpy()
-        checked = np.allclose(out, expected_val[0][0]) or np.allclose(out, expected_val[0][1])
```

Comment:
I didn't get the point of why needed to check two values here so removed another one.

Reply:
There was no deterministic results with torch.argmax